src/OmnichannelChatSDK.ts

```diff
@@ -459,7 +460,7 @@ class OmnichannelChatSDK {
 
         if (this.authSettings) {
             if (!this.authenticatedUserToken) {
-                await this.setAuthTokenProvider(this.chatSDKConfig.getAuthToken);
+                await this.setAuthTokenProvider(this.chatSDKConfig.getAuthToken, {throwError: true});
```

Comment:
why we need to call setAuthTokenProvider when we know the token is missing?, why not to throw the exception already>?

Reply:
It's a retry in case `getAuthToken()` fails intermittently